test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -113,42 +115,24 @@ test.describe(
 			] );
 
 			// Check the block in the canvas.
-			await editor.page.pause();
-			await navBlockUtils.selectNavigationItemOnCanvas( 'Custom link' );
+			await expect(
+				editor.canvas.locator(
+					`role=textbox[name="Navigation link text"i] >> text="Custom link"`
+				)
+			).toBeVisible();
```

Comment:
Interesting that this is already reported as flaky (https://github.com/WordPress/gutenberg/issues/48033). We might want to take a look at this test 🙂 .

Reply:
you are right, I'll have a look tomorrow!